sale_delivery_split_date/models/stock_picking.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2018 Alex Comba - Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class StockPicking(models.Model):
+
+    _inherit = "stock.picking"
+
+    min_dt = fields.Date(
```

Comment:
Is this field still needed? Why don't we directly use `scheduled_date`?
I see the definition of this field is almost unchanged from v8, but in that time the `date` field was not stored so this new `stored` field would have been needed _for filter purpose_.
Now `scheduled_date` is stored so it would be possible to use that for filters; what do you think?